lib/api/controllers/securityController.js

```diff
@@ -683,14 +698,14 @@ function mDelete (kuzzle, type, request) {
     throw new BadRequestError(`Number of delete to perform exceeds the server configured value (${kuzzle.config.limits.documentsWriteCount})`);
   }
 
-  return Promise.all(request.input.body.ids.map(id => {
+  return Bluebird.all(request.input.body.ids.map(id => {
     const deleteRequest = new Request({
       controller: 'security',
       action: `delete${action}`,
       _id: id
     }, request.context);
 
-    return new Promise(resolve => {
+    return new Bluebird(resolve => {
       kuzzle.funnel.getRequestSlot(deleteRequest, overloadError => {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this function to use "return" consistently. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3801)